test/fulfillment_order_test.rb

```diff
@@ -57,6 +57,21 @@ def setup
       end
     end
 
+    context "#locations_for_move" do
+      should "be able to list locations for a fulfillment order" do
+        fulfillment_order = ShopifyAPI::FulfillmentOrder.find(519788021)
+        fake "fulfillment_orders/#{fulfillment_order.id}/locations_for_move", method: :get,
+             body: load_fixture('locations')
+
+        locations = fulfillment_order.locations_for_move
+
+        assert_equal 2, locations.count
+        location = locations.first
+        assert location.is_a?(ShopifyAPI::Location)
```

Comment:
Similar to the fixture naming, the response are fulfillment_order_move_locations which each have a location. It could be modelled as a FulfillmentOrderMoveLocation with a `.location` which is a `Location`.

Reply:
Good point! It should be another object which contains a `Location` object. Updated.